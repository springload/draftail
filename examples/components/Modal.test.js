import React from "react";
import { shallow } from "enzyme";

import Modal from "./Modal";

describe("Modal", () => {
  it("has defaults", () => {
    expect(shallow(<Modal>Test</Modal>)).toMatchSnapshot();
  });

  it("forwards its props", () => {
    expect(
      shallow(<Modal forwarded>Test</Modal>)
        .find("Modal")
        .prop("forwarded"),
    ).toBe(true);
  });
});
