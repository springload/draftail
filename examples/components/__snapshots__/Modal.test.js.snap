// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal has defaults 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="modal__container--open"
  className={
    Object {
      "afterOpen": "modal--open",
      "base": "modal",
      "beforeClose": "modal--before-close",
    }
  }
  closeTimeoutMS={0}
  isOpen={false}
  overlayClassName={
    Object {
      "afterOpen": "modal__overlay--open",
      "base": "modal__overlay",
      "beforeClose": "modal__overlay--before-close",
    }
  }
  parentSelector={[Function]}
  portalClassName="portal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  Test
</Modal>
`;
